src/components/formik-forms/formik-radio-button.jsx

```diff
@@ -0,0 +1,107 @@
+const classNames = require('classnames');
+const PropTypes = require('prop-types');
+const React = require('react');
+import {Field} from 'formik';
+
+const FormikInput = require('./formik-input.jsx');
+
+require('./formik-radio-button.scss');
+require('../forms/row.scss');
+
+const FormikRadioButtonSubComponent = ({
+    buttonValue,
+    children,
+    className,
+    field,
+    label,
+    labelClassName,
+    ...props
+}) => (
+    <React.Fragment>
+        <input
+            checked={buttonValue === field.value}
+            className={classNames(
+                'formik-radio-button',
+                className
+            )}
+            name={field.name}
+            type="radio"
+            value={buttonValue}
+            onBlur={field.onBlur} /* eslint-disable-line react/jsx-handler-names */
+            onChange={field.onChange} /* eslint-disable-line react/jsx-handler-names */
+            {...props}
+        />
+        {label && (
+            <label
+                className={classNames(
+                    'formik-radio-label',
+                    labelClassName
+                )}
+                htmlFor={buttonValue}
```

Comment:
@benjiwheeler sorry for the super random comment, and I know this was already merged, but I think this isn't the correct use of the `htmlFor` attribute. `htmlFor` is the react version of the `for` attribute, which only refers to inputs by `id` [(src)](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#Attributes). The result of this (I think) would be that clicking on the label will not toggle the radio button the way it is should. You could probably use `field.name` for the id and htmlFor instead?

Reply:
Not random, and thanks for pointing this out! I confess to some sloppy cut and paste, and should have learned what the heck `htmlFor` does! :) Fixing this in https://github.com/LLK/scratch-www/pull/3250 .